homework-04/config/config.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I'd rather move all these constants to .env file and install https://www.npmjs.com/package/dotenv package

Reply:
fixed

Reply:
Please, create `.env.example` file. I can see .env is gitignored

Reply:
resolved.